sdk/storage/azure-storage-blob/src/main/java/com/azure/storage/blob/sas/BlobServiceSasSignatureValues.java

```diff
@@ -129,7 +143,12 @@ public BlobServiceSasSignatureValues() {
      * @param contentEncoding The content-encoding header for the SAS.
      * @param contentLanguage The content-language header for the SAS.
      * @param contentType The content-type header for the SAS.
+     * @deprecated Please use {@link #BlobServiceSasSignatureValues(String)},
+     * {@link #BlobServiceSasSignatureValues(OffsetDateTime, BlobSasPermission)}, or
+     * {@link #BlobServiceSasSignatureValues(OffsetDateTime, BlobContainerSasPermission)}
+     * followed by calls to the desired setters.
      */
+    @Deprecated
```

Comment:
No it would not be possible currently unless the user were to use the BlobSasImplUtil class in the implementation.util package (which is not a thing we should expect our customers to do)